runner/src/provisioner/provisioner.ts

```diff
@@ -108,14 +105,10 @@ export default class Provisioner {
   async scheduleLogPartitionJobs (userName: string, databaseName: string, schemaName: string): Promise<void> {
     await wrapError(
       async () => {
-        const userDbConnectionParameters = await this.hasuraClient.getDbConnectionParameters(userName);
-        const userCronPgClient = new this.PgClient({
-          user: userDbConnectionParameters.username,
-          password: userDbConnectionParameters.password,
-          database: this.config.cronDatabase,
-          host: this.config.hasuraHostOverride ?? userDbConnectionParameters.host,
-          port: this.config.hasuraPortOverride ?? userDbConnectionParameters.port,
-        });
+        const userDbConnectionParameters = await this.getPostgresConnectionParameters(userName);
+        userDbConnectionParameters.database = this.config.cronDatabase;
```

Comment:
```suggestion
        const userDbConnectionParameters = {
          ...(await this.getPostgresConnectionParameters(userName)),
          database: this.config.cronDatabase
        };
```

Nit: In this case it's probably safe to mutate, but it's generally a better idea to avoid it and create new objects.